xml/SchemaValidation/ilias_copa_9.xsd

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema
+        xmlns='http://www.w3.org/2001/XMLSchema'
+        xmlns:t="http://www.ilias.de/xml/Modules/ContentPage/copa"
+        targetNamespace="http://www.ilias.de/xml/Modules/ContentPage/copa"
+        elementFormDefault="qualified"
+>
+
+    <import schemaLocation="ilias_ds_4_3.xsd"/>
+
+    <!-- t:Data type -->
+    <complexType name='Data' mixed='true'>
+    </complexType>
+
+    <!-- ilObjContentPage -->
+    <element name="Copa">
+        <complexType>
+            <sequence>
+                <element ref="t:id" minOccurs="1" maxOccurs="1"/>
+                <element ref="t:title" minOccurs="1" maxOccurs="1"/>
+                <element ref="t:description" minOccurs="1" maxOccurs="1"/>
+                <element ref="t:info-tab" minOccurs="1" maxOccurs="1"/>
+            </sequence>
+        </complexType>
+    </element>
+
+    <element name="id" type="t:Data" />
+    <element name="title" type="t:Data" />
+    <element name="description" type="t:Data" />
+    <element name="info-tab" type="t:Data" />
```

Comment:
You removed the `style-id` Are you sure this is not exported anymore?

Reply:
it was at least never included in the exports I created.

I could also not find it anywhere else in the code except some other Modules/Services that still had it in the switch case.

Reply:
If it is no more part of the XML exporter, it is not used anymore.

But as far as I can see, the `Services/Style` component is still used a sa `Tail Dependecy` (see: `\ilContentPageExporter::getXmlExportTailDependencies`), so it might be included in the export and parsed during the import.

Reply:
but don't those end up in their own folder in the exported archive?

From **manifest.xml**:
```<?xml version="1.0" encoding="utf-8"?><!--Generated by ILIAS XmlWriter--><Manifest MainEntity="copa" Title="bgggg" InstallationId="0" InstallationUrl="http://ilias8"><ExportFile Component="Modules/ContentPage" Path="Modules/ContentPage/set_1/export.xml"/><ExportFile Component="Services/MediaObjects" Path="Services/MediaObjects/set_1/export.xml"/><ExportFile Component="Services/MetaData" Path="Services/MetaData/set_1/export.xml"/><ExportFile Component="Modules/File" Path="Modules/File/set_1/export.xml"/><ExportFile Component="Services/MetaData" Path="Services/MetaData/set_2/export.xml"/><ExportFile Component="Services/Object" Path="Services/Object/set_1/export.xml"/><ExportFile Component="Services/COPage" Path="Services/COPage/set_1/export.xml"/><ExportFile Component="Services/Style" Path="Services/Style/set_1/export.xml"/><ExportFile Component="Services/Object" Path="Services/Object/set_2/export.xml"/></Manifest>```

Reply:
I can get a different component (LearningModule) to include the StyleId in the main export.xml

but neither Chatroom, Forum nor ContentPage export that value for me.
And the LearningModule classes had some other mentions of "StyleId" in the code.